package.json

```diff
@@ -40,6 +40,7 @@
     "release:dry-run": "npm run release -- --dry-run"
   },
   "devDependencies": {
+    "@types/node": "^12.7.3",
```

Comment:
To fix "TS2300: Duplicate identifier 'IteratorResult'."

See also https://github.com/microsoft/TypeScript/issues/32333